app/views/articles/_actions.html.erb

```diff
@@ -5,23 +5,23 @@
       <%= render partial: "articles/reaction_button",
                  locals: {
                    category: :like,
-                   description: "Heart",
+                   description: t("core.heart"),
```

Comment:
I think we should eventually start using namespaced keys more, I'd suggest `core.reactions.heart`, etc.